styles/nav.scss

```diff
@@ -121,7 +121,7 @@ $daptiv-subnav-tab-h-pad: 14px;
 
     .daptiv-subnav-tab {
         color: $daptiv-subnav-font-color;
-        display: inline-block;
+        cursor: pointer;
```

Comment:
@joebri this should not be needed. We are leaving the cursor to whatever the browser default is.